docs/glab_mr.md

```diff
@@ -21,6 +21,7 @@ glab mr <command> [flags]
 
 * [glab](glab.md)	 - A GitLab CLI Tool
 * [glab mr approve](glab_mr_approve.md)	 - Approve merge requests
+* [glab mr approvers](glab_mr_approvers.md)	 - List merge requests eligible approvers
```

Comment:
The docs should not be edited. Just add the command, the docs will be automatically generated when PR is merged. Same applies to docs/glab_mr_approvers.md... Just focus on adding your feature 👍

Reply:
Cool, I'll just remove that commit again.